wiznotes/src/Components/AuthForms/AuthForms.css

```diff
@@ -0,0 +1,82 @@
+.main {
+    height: 100vh;
+    margin-top: 0rem;
+}
+
+.auth-card {
+    display: flex;
+    margin-bottom: 4rem;
+    margin-top: 5rem;
+    flex-direction: column;
```

Comment:
suggestion: you can use utilities from your component library here

Reply:
Yes will spacing in utilities then add here